R/sql-expr.R

```diff
@@ -77,3 +77,8 @@ replace_expr <- function(x, con) {
   }
 
 }
+
+# copied from DBI
+is_raw_list <- function(x) {
```

Comment:
Shouldn't this be using [blob](https://github.com/tidyverse/blob)?

Reply:
Updated to use blob instead of list of raw.